packaging/centos7/cloud.spec

```diff
@@ -482,7 +488,7 @@ pip install --upgrade /usr/share/cloudstack-marvin/Marvin-*.tar.gz
 %dir %attr(0770,root,cloud) %{_localstatedir}/cache/%{name}/management/work
 %dir %attr(0770,root,cloud) %{_localstatedir}/cache/%{name}/management/temp
 %dir %attr(0770,root,cloud) %{_localstatedir}/log/%{name}/management
-%config(noreplace) %{_sysconfdir}/sysconfig/%{name}-management
```

Comment:
@vincentbernat thanks, I was not sure but found that /etc/default exists and is used by several packages on CentOS and it would keep paths uniform across distributions. Should I revert back to sysconfig for CentOS packages then?